pyproject.toml

```
@@ -34,4 +34,4 @@ ignore_missing_imports = true
 [tool.pylint]
 max-line-length = 120
 ignored-modules = "gi"
-disable = "global-statement,invalid-name"
+disable = "global-statement,invalid-name,C0114,C0115,C0116"
```

I am more a fan of using the full names.

```suggestion
disable = "global-statement,invalid-name,missing-module-docstring,missing-class-docstring,missing-function-docstring"
```